src/role_atkeeper.js

```diff
@@ -9,17 +9,12 @@
 
 roles.atkeeper = {};
 
-roles.atkeeper.getPartConfig = function(room, energy, heal) {
-  var parts = [MOVE, RANGED_ATTACK, MOVE, HEAL];
-  return room.getPartConfig(energy, parts);
-};
-
-roles.atkeeper.energyRequired = function(room) {
-  return Math.min(room.getEnergyCapacityAvailable(), 3430);
-};
-
-roles.atkeeper.energyBuild = function(room) {
-  return Math.min(room.getEnergyCapacityAvailable(), 3430);
+roles.atkeeper.getPartConfig = function(room) {
+  let datas = {
```

Comment:
remove inline variable ?
a +1 for move datas to config 
e.g. config.roles.atkeeper or maybe config.creep.layouts.atkeeper or something like that